utils/schemavalidation/schemavalidation.go

```diff
@@ -0,0 +1,49 @@
+package schemavalidation
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/pkg/errors"
+	"github.com/qri-io/jsonschema"
+)
+
+// Validate validates the provided objectBytes against the provided CNAB-Spec schemaType
+// The schemaType must be a valid schema currently hosted at https://cnab.io/v1/*.schema.json
+func Validate(schemaType string, objectBytes []byte) ([]jsonschema.ValError, error) {
+	url := fmt.Sprintf("https://cnab.io/v1/%s.schema.json", schemaType)
+	req, err := http.NewRequest("GET", url, nil)
```

Comment:
I don't know if it's a good thing to make that http request upon validation, as it will fail in airgap environment.
And I also have some concerns with tests depending on external resources (so it might break all of a sudden without knowing why). Maybe we should "vendor" those files in cnab-go ?

Reply:
That's a good point.
We should ensure we keep the schemas in sync with whatever CNAB version we're implementing here, and it may be a great opportunity to add tests that validate all bundles we produce against the local schemas as well (not in this PR though).

Reply:
Indeed, we'll need to look for a more robust solution.  I'll mark this PR as WIP for now.  I've started a convo in the CNAB Slack channel with ideas...

Reply:
Updated this PR with an approach that injects the spec versions at build-time in addition to fetching the actual schema files and baking them in.. PTAL when convenient.